lib/agent0/agent0/accounts_config.py

```diff
@@ -0,0 +1,190 @@
+"""Defines the accounts configuration from env vars."""
+from __future__ import annotations
+
+import json
+import logging
+import os
+import sys
+from dataclasses import dataclass
+
+import numpy as np
+from dotenv import load_dotenv
+from fixedpointmath import FixedPoint
+
+from .base.config import AgentConfig, Budget
+from .base.make_key import make_private_key
+
+
+@dataclass
+class AccountKeyConfig:
+    """The account config dataclass
+
+    Attributes
+    ----------
+    ARTIFACTS_URL: str
+        The url of the artifacts server from which we get addresses.
+    RPC_URL: URI | str
```

Comment:
I guess all of these should be URI and not URL right?

Reply:
Won't fix, these env variables match what's in infra: https://github.com/delvtech/infra/blob/main/env/env.common